common/response.go

```diff
@@ -66,6 +67,7 @@ type Response struct {
 	url               string
 	status            int64
 	statusText        string
+	bodyMx            sync.RWMutex
```

Comment:
Let's use the conventional `bodyMu` name instead of `bodyMx`.

Reply:
I wasn't aware `Mu` was a convention, I've seen both usages. And kind of prefer `Mx` because it's short for **m**utual e**x**clusion. But sure, `Mu` is more common in this project, so I'll change it.